build/wix/subdirs.txt

```diff
@@ -0,0 +1,7 @@
+controllers
```

Comment:
Damned, I modified the bat files and forgot to remove this now unneeded txt file.